Cognite.Simulator.Extensions/SequencesExtensions.cs

```diff
@@ -157,20 +157,29 @@ public static class SequencesExtensions
         /// <param name="sequenceExternalId">Simulator integration sequence external id</param>
         /// <param name="update">Data to be updated, if init is set to true</param>
         /// <param name="token">Cancellation token</param>
+        /// <param name="updateLicense"></param>
         /// <exception cref="SimulatorIntegrationSequenceException">Thrown when one or more sequences
         /// rows could not be updated. The exception contains the list of errors</exception>
         public static async Task UpdateSimulatorIntegrationsHeartbeat(
             this SequencesResource sequences,
             string sequenceExternalId,
             bool init,
             SimulatorIntegrationUpdate update,
-            CancellationToken token)
+            CancellationToken token,
+            bool updateLicense = false)
         {
             var rowsToCreate = new List<SequenceDataCreate>();
-            var rowData = new Dictionary<string, string>
-                {
-                    { SimulatorIntegrationSequenceRows.Heartbeat, $"{DateTime.UtcNow.ToUnixTimeMilliseconds()}" }
-                };
+            var rowData = new Dictionary<string, string>();
+
+            if (updateLicense is true)
+            {
+                rowData.Add(SimulatorIntegrationSequenceRows.LicenseTimestamp, $"{DateTime.UtcNow.ToUnixTimeMilliseconds()}");
```

Comment:
TODO: Convert to separate functions. Also consider renaming to make it more clear

Reply:
Ensure row is going to right column